### PR TITLE
feat: add locale(th-TH)

### PR DESCRIPTION
**Pull request by dean16930**
feat:

- Added Thai language pack
- Modified Locale README

todo:

 Need to be merged into V3

### CODE REVIEW

File: src/locale/README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Current supported languages:
 | Romanian                 | ro-RO        |
 | Spanish (Spain)          | es-ES        |
 | Turkish                  | tr-TR        |
+| Thai                     | th-TH        |
```
Comment (chenjiahan) [inline src/locale/README.md:55]: `locale/README.zh-CN.md` should also be updated

Comment (dean16930) [reply src/locale/README.md:55]: ok

**Review by chenjiahan — APPROVED** [review]

**Comment by chenjiahan** [comment]
👏👏👏 Can you make a same Pull Request to dev (v3) branch?

**Comment by chenjiahan** [comment]
> 👏👏👏 Can you make a same Pull Request to dev (v3) branch?

Merged into v3: https://github.com/youzan/vant/pull/8297

**Comment by y1324** [comment]
请问V2版本有泰国对应翻译吗？

**Comment by chenjiahan** [comment]
有

**Comment by y1324** [comment]
> 有

请问下，sku里点击购买数量的增减按钮的提示语的翻译从哪里更改？还有不选商品规格的时候，点击立即购买按钮的提示语的翻译从哪里更改？

**Comment by chenjiahan** [comment]
https://vant-contrib.gitee.io/vant/#/zh-CN/locale#yu-yan-bao-zhong-bu-bao-han-sku-zu-jian

**Comment by y1324** [comment]
> https://vant-contrib.gitee.io/vant/#/zh-CN/locale#yu-yan-bao-zhong-bu-bao-han-sku-zu-jian

 "vant": "^2.10.8",  这个版本没有泰文的翻译文件。

**Comment by chenjiahan** [comment]
请看更新日志。

**Pull request merged**